kore/src/Kore/Rewrite/SMT/Evaluator.hs

```diff
@@ -152,15 +133,14 @@ filterMultiOr ::
     MultiOr (Conditional variable term) ->
     simplifier (MultiOr (Conditional variable term))
 filterMultiOr multiOr = do
-    elements <- mapM refute (toList multiOr)
-    return (MultiOr.make (catMaybes elements))
+    elements <- mapM refute $ toList multiOr
+    return $ MultiOr.make $ catMaybes elements
```

Comment:
I don't think we should, in general, remove parenthesis and replace them with `$` or `&`.

IMO, `$` and `&` are good when you have a larger block of code you want to split up and indent nicely.

I mean, we should decide which to use depending on what is most readable in the particular context. What do you think?

Reply:
Reverted: https://github.com/kframework/kore/pull/2864/commits/cfe85ebac7b672ce17b69fd812b7a9b7b987f15f